# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../runit/patches/ntp-etc_ntp.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./sntp/header.h.orig	2005-02-04 16:50:35.587660840 -0300
+++ ./sntp/header.h	2005-02-04 16:53:34.722428216 -0300
@@ -19,10 +19,10 @@
 #define MAX_SOCKETS        10          /* Maximum number of addresses */
 
 #ifndef LOCKNAME
-    #define LOCKNAME "/etc/msntp.pid"  /* Stores the pid */
+    #define LOCKNAME "/etc/ntp/msntp.pid"  /* Stores the pid */
 #endif
 #ifndef SAVENAME
-    #define SAVENAME "/etc/msntp.state" /* Stores the recovery state */
+    #define SAVENAME "/etc/ntp/msntp.state" /* Stores the recovery state */
 #endif
 
 
--- ./parseutil/dcfd.c.orig	2005-02-04 16:55:37.499763224 -0300
+++ ./parseutil/dcfd.c	2005-02-04 16:57:05.094446808 -0300
@@ -1390,7 +1390,7 @@
 	char **a = argv;
 	int  ac = argc;
 	char *file = NULL;
-	const char *drift_file = "/etc/dcfd.drift";
+	const char *drift_file = "/etc/ntp/dcfd.drift";
 	int fd;
 	int offset = 15;
 	int offsets = 0;
--- ./include/ntp_config.h.orig	2005-02-04 17:00:41.984474512 -0300
+++ ./include/ntp_config.h	2005-02-04 17:01:05.737863448 -0300
@@ -3,7 +3,7 @@
  */
 #ifndef CONFIG_FILE
 # ifndef SYS_WINNT
-#  define	CONFIG_FILE "/etc/ntp.conf"
+#  define	CONFIG_FILE "/etc/ntp/ntp.conf"
 # else /* SYS_WINNT */
 #  define	CONFIG_FILE	"%windir%\\system32\\drivers\\etc\\ntp.conf"
 #  define	ALT_CONFIG_FILE "%windir%\\ntp.conf"
--- ./libntp/audio.c.orig	2005-02-04 17:02:36.394081608 -0300
+++ ./libntp/audio.c	2005-02-04 17:02:58.902659784 -0300
@@ -53,7 +53,7 @@
 static struct audio_device device; /* audio device ident */
 #endif /* HAVE_SYS_AUDIOIO_H */
 #ifdef PCM_STYLE_SOUND
-# define INIT_FILE "/etc/ntp.audio"
+# define INIT_FILE "/etc/ntp/ntp.audio"
 int agc =	SOUND_MIXER_WRITE_RECLEV; /* or IGAIN or LINE */
 int monitor =	SOUND_MIXER_WRITE_VOLUME; /* or OGAIN */
 int devmask = 0;
--- ./ntpdate/ntpdate.c.orig	2005-02-04 17:04:14.381185304 -0300
+++ ./ntpdate/ntpdate.c	2005-02-04 17:04:38.009593240 -0300
@@ -183,7 +183,7 @@
 
 #ifndef KEYFILE
 # ifndef SYS_WINNT
-#define KEYFILE 	"/etc/ntp.keys"
+#define KEYFILE 	"/etc/ntp/ntp.keys"
 # else
 #define KEYFILE 	"%windir%\\ntp.keys"
 # endif /* SYS_WINNT */
--- ./ntpdate/ntptime_config.c.orig	2005-02-04 17:05:13.372217304 -0300
+++ ./ntpdate/ntptime_config.c	2005-02-04 17:06:05.934226664 -0300
@@ -37,7 +37,7 @@
  */
 #ifndef CONFIG_FILE
 # ifndef SYS_WINNT
-#  define	CONFIG_FILE "/etc/ntp.conf"
+#  define	CONFIG_FILE "/etc/ntp/ntp.conf"
 # else /* SYS_WINNT */
 #  define	CONFIG_FILE 	"%windir%\\ntp.conf"
 #  define	ALT_CONFIG_FILE "%windir%\\ntp.ini"
--- ./ntpdate/ntptimeset.c.orig	2005-02-04 17:06:29.360665304 -0300
+++ ./ntpdate/ntptimeset.c	2005-02-04 17:06:43.329541712 -0300
@@ -271,7 +271,7 @@
 
 #ifndef KEYFILE
 # ifndef SYS_WINNT
-#define KEYFILE 	"/etc/ntp.keys"
+#define KEYFILE 	"/etc/ntp/ntp.keys"
 # else
 #define KEYFILE 	"%windir%\\ntp.keys"
 # endif /* SYS_WINNT */
--- ./ntpd/refclock_oncore.c.orig	2005-02-04 17:10:10.164098080 -0300
+++ ./ntpd/refclock_oncore.c	2005-02-04 17:10:36.818046064 -0300
@@ -491,7 +491,7 @@
 
 #define DEVICE1 	"/dev/oncore.serial.%d"   /* name of serial device */
 #define DEVICE2 	"/dev/oncore.pps.%d"   /* name of pps device */
-#define INIT_FILE	"/etc/ntp.oncore" /* optional init file */
+#define INIT_FILE	"/etc/ntp/ntp.oncore" /* optional init file */
 
 #define SPEED		B9600		/* Oncore Binary speed (9600 bps) */
 
